scripts/split-e2e-tests.ts

```diff
@@ -334,13 +338,15 @@ function splitTests(
           if (typeof workflowJob === 'string') {
             return newJobName;
           } else {
+            // for the most up-to-date list of executors for e2e, see the config.base.yml file
+            let linuxVMSize = USE_LINUX_LARGE_VM.includes(newJobName) ? 'l_large' : 'l_medium';
             return {
               [newJobName]: {
                 ...Object.values(workflowJob)[0],
                 requires: [...(requires ? [requires] : workflowJob[jobName].requires || [])],
                 matrix: {
                   parameters: {
-                    os: WINDOWS_TEST_ALLOWLIST.includes(newJobName) ? ['l', 'w'] : ['l'],
+                    os: WINDOWS_TEST_ALLOWLIST.includes(newJobName) ? [linuxVMSize, 'w'] : [linuxVMSize],
```

Comment:
I would be a fan of changing this 'w' to something more descriptive 🙉

Reply:
just updated it!